rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -214,9 +214,9 @@ private byte getRealV(byte v) {
     // There was a method called NEW_getTransactionCost that implemented this alternative solution:
     // "return (this.isContractCreation() ? GasCost.TRANSACTION_CREATE_CONTRACT : GasCost.TRANSACTION)
     //         + zeroVals * GasCost.TX_ZERO_DATA + nonZeroes * GasCost.TX_NO_ZERO_DATA;"
-    public long transactionCost(long blockNumber, BlockchainNetConfig netConfig) {
+    public long transactionCost(Constants constants, ActivationConfig.ForBlock activationConfig) {
```

Comment:
Parameter rename (I'm sorry for being so repetitive, I think it may be useful so you don't have to review all the code again)